app-modules/notification/database/migrations/2023_12_20_160721_create_outbound_deliverables_table.php

```diff
@@ -44,20 +44,24 @@ public function up(): void
     {
         Schema::create('outbound_deliverables', function (Blueprint $table) {
             $table->uuid('id')->primary();
-            $table->string('related_id')->nullable();
-            $table->string('related_type')->nullable();
-            $table->string('recipient_id')->nullable();
-            $table->string('recipient_type')->nullable();
+
             $table->string('channel');
             $table->string('notification_class');
             $table->string('external_reference_id')->nullable()->unique();
             $table->string('external_status')->nullable();
             $table->json('content')->nullable();
             $table->string('delivery_status')->default(NotificationDeliveryStatus::Awaiting);
-            $table->timestamp('delivered_at')->nullable();
-            $table->timestamp('last_delivery_attempt')->nullable();
             $table->longText('delivery_response')->nullable();
             $table->integer('quota_usage')->default(0);
+
+            $table->string('related_id')->nullable();
+            $table->string('related_type')->nullable();
+            $table->string('recipient_id')->nullable();
+            $table->string('recipient_type')->nullable();
```

Comment:
I know our pattern is a bit scattered throughout the application, but is there a reason we're moving these columns?

Reply:
Because we don't have a pattern so I've been using my personal pattern which is: primary key, data columns, foreign keys, timestamps(custom, created/updated, deleted). So when I edit a migration I switch to that and add spaces between the groups.